airflow/utils/state.py

```diff
@@ -41,6 +41,21 @@ class State(object):
     UPSTREAM_FAILED = "upstream_failed"
     SKIPPED = "skipped"
 
+    task_states = (
+        SUCCESS,
+        RUNNING,
+        FAILED,
+        UPSTREAM_FAILED,
+        UP_FOR_RETRY,
+        QUEUED,
+    )
```

Comment:
We should make sure we sync those, because I feel like we have seen this issue internally where `SCHEDULED` tasks appear with a `None` state of sorts. It gets a little confusing.